docs/services/aws-services-guide.md

```diff
@@ -14,388 +14,55 @@ Crossplane managed resources and the official Wordpress Docker image.
 
```

Comment:
Suggest changing the main heading to "AWS Services Guide" to more close links to this as the AWS services guide, to more closely match the ToC title, and to be consistent with the GCP Services Guide.

Reply:
This was updated in most recent commit. Reference here is to commit prior :+1: